monkey/infection_monkey/utils/commands.py

```diff
@@ -0,0 +1,15 @@
+import shlex
+
+
+def get_monkey_cmd_lines_windows(monkey_cmdline_windows, destination_path, monkey_options):
```

Comment:
Maybe it makes sense for  `build_monkey_commandline()` and `build_monkey_commandlinu_explicitly()` to be moved from `helpers.py` to this file.